openstax_accounts/stub.py

```diff
@@ -183,23 +168,19 @@ def search(self, query, **kwargs):
                 'items': [],
                 'total_count': 0,
                 }
+        items = []
         for username in self.users:
             profile = self.users[username]['profile']
-            values = [username]
-            for key, value in profile.items():
-                if key == 'contact_infos':
-                    values.append(profile['contact_infos'][0]['value'])
-                else:
-                    values.append(value)
-
-            for value in values:
-                if fnmatch.fnmatch(username, query):
-                    results['items'].append({
-                        'username': username,
-                        'id': profile['id'],
-                        })
+            if profile is None:
+                profile = {'username': username}
+
+            for value in profile.values():
+                if fnmatch.fnmatch(username, query) \
+                   or username in query:
+                    items.append(profile)
                     break
 
+        results['items'] = list(items)
```

Comment:
Is there a reason to create a list and then copy it again to results['items']?  Is it not ok to either do `item = results['items']` on line 171 or just use `results['items']` directly?